src/Integration.Vsix/SonarLintDaemon/CFamily.cs

```diff
@@ -106,11 +153,11 @@ public static FileConfig TryGet(EnvDTE.ProjectItem projectItem, string absoluteF
             /// Computes property value taking into account inheritance, property sheets and macros.
             /// </summary>
             /// <param name="propertyName">name of the property (note that name corresponds to xml tag in vcxproj)</param>
-            private static string GetEvaluatedPropertyValue(dynamic fileTool, string propertyName)
+            private static string GetEvaluatedPropertyValue(object fileTool, string propertyName)
```

Comment:
Minor change - there was a hard cast to _object_ below, so it might as well be typed as _object_.